src/components/WatchOnButton.vue

```diff
@@ -19,11 +19,11 @@ export default {
         <!-- For large screens -->
         <a :href="link" class="btn flex items-center lt-lg:hidden">
             <i18n-t keypath="player.watch_on" tag="strong">{{ platform }}</i18n-t>
-            <font-awesome-icon class="mx-1.5" :icon="['fab', platform.toLowerCase()]" />
+            <i :class="'i-fa6-brands:' + platform.toLowerCase()" class="mx-1.5" />
```

Comment:
This is the only replacement that doesn't work and I'm unsure why.

1. i-fa6-brands:youtube is shown correctly in the DOM
2. Hardcoding this line to ```i-fa6-brands:youtube``` works correctly
3. Updating ```platform``` to another icon, i.e. Bitcoin, shows that icon correctly

Reply:
Actually, updating ```platform``` to Apple, or a brand that isn't already used, doesn't work either. I assume Bitcoin works because it is already cached from being used in the footer(?)

Reply:
Yeah, the preprocessor probably only "compiles" the icons that it can find in the plain source code, and since we're using dynamic icon names here it can't recognize it. We should likely just do a `v-if / v-else` block here for each possible platform out of YouTube and Odysee.